CHANGELOG.md

```diff
@@ -2,6 +2,24 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [1.3.0](https://github.com/onfido/castor/compare/v1.2.0...v1.3.0) (2021-05-12)
```

Comment:
It skipped the 1.2.1 and 1.2.2 changelog here, as expected. What did you do last time, manual copy over?

Reply:
No, I skipped it.

All fixes are placed within 1.3.0 log, which even not ideal I believe is fine... It still reflects changes from 1.2.0 to 1.3.0.